### PR TITLE
Add Blog Tutorial

### PR DESCRIPTION
**Pull request by hamelsmu**
# Changes Made

- Added Blogging Tutorial 
      - Standalone blogs -> Instruct people how to migrate from fastpages to Quarto (nbdev is not necessary)
      - Blogs in your nbdev site -> Instruct people how to add a blog to your nbdev project
- Created a proper blogging site for nbdev  (it is a sub-section within our current site) with two posts copied from fast.ai (Jupyter-Git Problem is solved & Announcing nbdev), The navbar now will point to this blog sub-section
- Made capitalization for page titles consistent in https://github.com/fastai/nbdev/pull/1061/commits/8739d0356736c49510a4d369ee037c017baf0a6a
- Refactored the Modular nbdev page to highlight areas better and to add `showdoc` in https://github.com/fastai/nbdev/pull/1061/commits/ced9bb6662519ceac7277b937819fa92db2791e6

Note: I got rid of the mention of `nbdev. quarto` from modular nbdev b/c it is not clear to me how it should be used outside nbdev, but happy to add it back if you want to. 

@seeM @jph00

### CODE REVIEW

**Comment by hamelsmu** [comment]
This PR will close https://github.com/fastai/nbdev/issues/649

File: nbs/_quarto.yml [hunk 1]
```diff
@@ -33,10 +33,10 @@ website:
     left:
       - text: "Get Started"
         href: getting_started.ipynb
-      - text: "Tutorial"
+      - text: "Tutorials"
```
Comment (seeM) [inline nbs/_quarto.yml:36]: This is a link to the actual tutorial, so `Tutorial` was correct. Unless we think linking to the tutorials index is better, then below line should become `href: tutorials`

Comment (hamelsmu) [reply nbs/_quarto.yml:36]: oh I see good catch, I misunderstood

File: nbs/blog/index.qmd [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (seeM) [inline nbs/blog/index.qmd:1]: This is very cool!

File: nbs/tutorials/modular_nbdev.ipynb [hunk 3]
```diff
@@ -1,5 +1,15 @@
 {
  "cells": [
+  {
+   "cell_type": "raw",
+   "id": "d3d59c9d-a2f2-4ee8-a05a-000d171aa01d",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "skip_showdoc: true\n",
```
Comment (seeM) [inline nbs/tutorials/modular_nbdev.ipynb:9]: Can you use the markdown frontmatter syntax here? I think we agreed at some point that its preferred over raw

Comment (hamelsmu) [reply nbs/tutorials/modular_nbdev.ipynb]: 👍🏽

**Comment by jph00** [comment]
This is great! lgtm

File: nbs/_quarto.yml (hunk 1 above)
Comment (hamelsmu) [reply nbs/_quarto.yml]: ok reverted this 👍🏽

**Pull request merged**